<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tuning Fork Fitting</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resources/nSOTScannerResources.qrc">
    <normaloff>:/nSOTScanner/Pictures/jj_schema.svg</normaloff>:/nSOTScanner/Pictures/jj_schema.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget
{
background-color: rgb(0, 0, 0);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1" columnstretch="1,1,1,1">
    <item row="2" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_Gain">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Multiply the Data before doing fit help with better accuracy</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#lineEdit_Gain{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color:rgb(131,131,131)</string>
      </property>
      <property name="text">
       <string>Method</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton_Ok">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Test</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed#pushButton_Ok{
color: gray;
background-color:rgb(168,168,168);
border: 1px solid rgb(95,107,166);
border-radius: 5px
}

QPushButton#pushButton_Ok{
color: gray;
background-color:rgb(0,0,0);
border: 2px solid gray;
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label_Feedback">
      <property name="styleSheet">
       <string notr="true">color:gray</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#838383;&quot;&gt;Feedback&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color:rgb(131,131,131)</string>
      </property>
      <property name="text">
       <string>Gain</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit_Amplitude">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Amplitude fo Tuning Fork</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#lineEdit_Amplitude{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_Angle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>North is zero, clockwise is positive</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#lineEdit_Angle{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color:rgb(131,131,131)</string>
      </property>
      <property name="text">
       <string>nm</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color:rgb(131,131,131)</string>
      </property>
      <property name="text">
       <string>degree</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QComboBox" name="comboBox_Method">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Method</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox#comboBox_Method{
color: rgb(131,131,131);
background-color:rgb(0,0,0);
border: 2px solid  rgb(131,131,131);
border-radius: 5px
}</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Nelder-Mead</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Powell</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BFGS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Newton-CG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>L-BFGS-B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TNC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COBYLA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SLSQP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>trust-constr</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dogleg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>trust-ncg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>trust-exact</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>trust-krylov</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
